harness/determined/transformers/_hf_callback.py

```diff
@@ -287,6 +287,10 @@ def _log_config_mismatch(
             f"use (--num_train_epochs and searcher.max_length.epochs) OR "
             f"(--max_steps and searcher.max_length.batches)."
         )
+        raise ValueError(
+            f"HF trainer units {trainer_units}={trainer_len} do not match searcher config "
```

Comment:
Being nitpicky from the hand-off call: maybe "HF trainer units MUST match search config..." and if we can, a blurb about how to address this just to make it clearly actionable, not just a description of the problem.

Reply:
yeah makes sense. we had a message in the previous error log that elaborated a bit on how to address this issue, but i moved it to the exception now. thanks!